Failed to deploy sample with `make sample`

Error:

>Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "mharbor.kb.io": the server could not find the requested resource
Makefile:117: recipe for target 'sample' failed
make: *** [sample] Error 1

Check endpoints of webhook SVC:

```yaml
kubectl describe service -n harbor-operator-system harbor-operator-webhook-service
Name:              harbor-operator-webhook-service
Namespace:         harbor-operator-system
Labels:            <none>
Annotations:       kubectl.kubernetes.io/last-applied-configuration:
                     {"apiVersion":"v1","kind":"Service","metadata":{"annotations":{},"name":"harbor-operator-webhook-service","namespace":"harbor-operator-sys...
Selector:          control-plane=controller-manager
Type:              ClusterIP
IP:                10.96.115.134
Port:              <unset>  443/TCP
TargetPort:        9443/TCP
Endpoints:         10.244.1.12:9443
Session Affinity:  None
Events:            <none>
```

--- a/api/v1alpha1/harbor_webhook.go
+++ b/api/v1alpha1/harbor_webhook.go
@@ -15,7 +15,7 @@ func (r *Harbor) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		Complete()
 }
 
-// +kubebuilder:webhook:path=/mutate-goharbor-v1alpha1-harbor,mutating=true,failurePolicy=fail,groups=goharbor.io,resources=harbors,verbs=create;update,versions=v1alpha1,name=mharbor.kb.io
+// +kubebuilder:webhook:path=/mutate-goharbor-io-v1alpha1-harbor,mutating=true,failurePolicy=fail,groups=goharbor.io,resources=harbors,verbs=create;update,versions=v1alpha1,name=mharbor.kb.io
 
 var _ webhook.Defaulter = &Harbor{}
 

